src/start_package/mod.rs

```diff
@@ -1,3 +1,4 @@
+use kinode_process_lib::kernel_types::Erc721Metadata;
```

Comment:
Would prefer this be in a third grouping of imports after the one starting with `serde_json`. The general convention I like to follow is:

1. Alphabetized within a group.
2. First group: "stdlibs"
3. Second group: imported libs/crates
4. Third group: our libs/crates

and so on: from more "general"/widely distributed to more "specific"/local.

Reply:
Sorry, one more down in its own grp, e.g.

```
use std::foo;
...

use serde_json::bar;
...

use kinode_process_lib::baz;
...

use super::inject_message;
...
```